hepcrawl/extractors/iop_parser.py

```diff
@@ -124,3 +120,47 @@ def get_arxiv_eprints(self, node):
             arxiv_eprints.append({'value': pattern.sub('', arxiv_value)})
 
         return arxiv_eprints
+
+    def get_license_type_and_version_from_url(self, url):
+        LICENSE_PATTERN = re.compile(r"([A-Za-z]+)\/([0-9]+\.[0-9]+)")
+        CREATIVE_COMMONS_PATTERN = re.compile(r".*(creative\s{0,}commons).*", flags=re.I)
```

Comment:
we don't need to compile them every time we call the function. Let's put them on the module level.